core/server/helpers/tpl/navigation.hbs

```diff
@@ -1,5 +1,5 @@
 <ul class="nav">
     {{#foreach navigation}}
-    <li class="nav-{{slug}}{{#if current}} nav-current{{/if}}" role="presentation"><a href="{{url absolute="true"}}">{{label}}</a></li>
+    <li class="nav-{{slug}}{{#if current}} nav-current{{/if}}" role="presentation"><a href="{{{url absolute="true"}}}">{{label}}</a></li>
```

Comment:
I had a quick look into how to remove the triple-curlies here, I think it would rely on encoding within the `{{url}}` helper and marking the string has html-safe however I wasn't sure how that would affect things elsewhere.